doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -288,6 +288,24 @@ Configure radio transmitter power
 
 See :ref:`ug_matter_gs_transmission_power` for more information.
 
+Configure radio driver temperature calibration period
+*****************************************************
+
+The radio driver requires periodic calibration events to compensate the potential temperature changes that may affect clocks configuration.
+How often the calibration should take place depends on the device's environment and what temperature changes the device is exposed to.
+For example, a device that is mobile or installed outdoor likely requires smaller calibration period than a device that is installed indoor.
+
+Additionally, the calibration period has impact on the device's power consumption, because it leads to the device wake-ups.
+It is recommended to not use calibration period value smaller than required to avoid the unnecessary power consumption increase.
```

Comment:
```suggestion
It is recommended not to use a calibration period value smaller than required to avoid the unnecessary increase in power consumption.
```